PWGCF/FemtoUniverse/Tasks/femtoUniversePairTaskTrackV0Extended.cxx

```diff
@@ -268,6 +271,21 @@ struct femtoUniversePairTaskTrackV0Extended {
       pairCloseRejection.init(&resultRegistry, &qaRegistry, ConfCPRdeltaPhiCutMin.value, ConfCPRdeltaPhiCutMax.value, ConfCPRdeltaEtaCutMin.value, ConfCPRdeltaEtaCutMax.value, ConfCPRChosenRadii.value, ConfCPRPlotPerRadii.value);
       pairCloseRejectionV0.init(&resultRegistry, &qaRegistry, ConfCPRdeltaPhiCutMin.value, ConfCPRdeltaPhiCutMax.value, ConfCPRdeltaEtaCutMin.value, ConfCPRdeltaEtaCutMax.value, ConfCPRChosenRadii.value, ConfCPRPlotPerRadii.value);
     }
+
+    if (!ConfLocalEfficiency.value.empty()) {
+      plocalEffFile = std::unique_ptr<TFile>(TFile::Open(ConfLocalEfficiency.value.c_str(), "read"));
+      if (!plocalEffFile || plocalEffFile.get()->IsZombie())
+        LOGF(fatal, "Could not load efficiency histogram from %s", ConfLocalEfficiency.value.c_str());
+      if (doprocessSameEvent || doprocessMixedEvent) {
+        plocalEffp1 = (ConfChargePart1 > 0) ? std::unique_ptr<TH1>(plocalEffFile.get()->Get<TH1>("PrPlus")) : std::unique_ptr<TH1>(plocalEffFile.get()->Get<TH1>("PrMinus")); // note: works only for protons for now
```

Comment:
For the future: you don't need to use `get()` on unique_ptr. The `->` operator will already reference to the underlying object, in the same way it works with the raw pointers.

Reply:
so `plocalEffFile->Get<TH1>("PrPlus")` would also work

Reply:
Thanks a lot for the tips.